R/multisynth_class.R

```diff
@@ -39,13 +37,12 @@ multisynth <- function(form, unit, time, data,
     trt <- terms(formula(form, rhs=1))[[3]]
     wide <- format_data_stag(outcome, trt, unit, time, data)
 
-    
-    
-    ## if n_leads is NULL set it to be the largest possible number of leads
-    if(is.null(n_leads)) {
-        n_leads <- max(apply(1-wide$mask, 1, sum)) + ncol(wide$y)
-    } else if(n_leads > max(apply(1-wide$mask, 1, sum)) + ncol(wide$y)) {
-        n_leads <- max(apply(1-wide$mask, 1, sum)) + ncol(wide$y)
+
+    ## if n_leads is NULL set it to be the largest possible number of leads where all treatment units still have data  
+    if(is.null(n_leads) || n_leads > max(apply(1-wide$mask, 1, sum)) + ncol(wide$y)) {
+        # n_leads <- max(apply(1-wide$mask, 1, sum)) + ncol(wide$y)
+        n_leads <- dim(wide$y)[2]
+        print(n_leads)
```

Comment:
Drop print statement? 

Also I think I forgot you were doing this change here and changed it to be like this in master, hence the merge conflict.

Reply:
My bad about the print statement; good catch.  Let me know if there's a better place to do this in; I just thought that `multisynth_class` was the most natural place.

Reply:
Eh, it currently exists in master, so just use the master version when you merge.